dataactvalidator/config/sqlrules/fabs43_detached_award_financial_assistance_3.sql

```diff
@@ -28,6 +31,7 @@ WHERE CASE WHEN COALESCE(dafa.place_of_performance_congr, '') <> ''
                         FROM state_congressional AS sc_2
                         WHERE UPPER(LEFT(dafa.place_of_performance_code, 2)) = sc_2.state_code) < 2
                 )
+                OR (dafa.record_type = 3)
```

Comment:
I don't know how the planner does things, but just in case there IS short-circuiting and the planner doesn't reorganize our code, we might want to put this as the top case inside this THEN

Reply:
Agreed. Will reorganize.